automation/test.sh

```diff
@@ -432,7 +432,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   fi
 fi
 
-if [[ $KUBEVIRT_NONROOT =~ true ]]; then
+if [[ $FEATURE_GATES != *"Root"* ]]; then
```

Comment:
Can't we drop `(verify-non-root)` and `decorators.NonRoot`?

Reply:
We sure can.
I think the meanings will be:
1. The condition could be removed from `automation/test.sh`.
2. The nonroot test should have a skip if the Root FG is enabled, so it won't fail on rootful lanes.
@xpivarc WDYT?

Reply:
For the 2nd point, I would let them fail. Users can skip them. This is IMHO much better than a silent skip

Reply:
What decorator are we talking about? But I agree skipping on the label is better.

Reply:
The `decorators.NonRoot` one! I think we still need it to exclude the non root tests from the root lanes

Reply:
Oh, yes, 100%

Reply:
Please note that this decorator is only used once, in this test.

Reply:
Yes, in the `automation/test.sh` script we only use the `--label-filter` ginkgo arg.
So without it we are not able to skip it, except with the explicit `Skip` in the code.

Reply:
@fossedihelm @xpivarc I think we can remove the decorator if it is not used anywhere else.
The alternative to removing the label is a replacing the assertion with a skip in the code, which we agree is not a good practice.
Do you agree?